whois-api/src/test/java/net/ripe/db/whois/api/rdap/WhoisRdapServiceTestIntegration.java

```diff
@@ -1492,43 +1493,12 @@ public void search_nameservers_not_found() {
                     .request(MediaType.APPLICATION_JSON_TYPE)
                     .get(Entity.class);
             fail();
-        } catch (NotFoundException e) {
-            assertErrorTitle(e, "nameservers not found");
-        }
-    }
-
-    @Test
-    public void search_nameservers_empty_name() {
-        try {
-            fullTextIndex.rebuild();
-            createResource("nameservers?name=")
-                    .request(MediaType.APPLICATION_JSON_TYPE)
-                    .get(Entity.class);
-            fail();
-        } catch (BadRequestException e) {
-            assertErrorTitle(e, "empty lookup key");
+        } catch (ServerErrorException e) {
```

Comment:
Can you add in a test for nameservers (i.e. the search) - the previous ones were removed

Reply:
Thanks Mahesh